docs/source/configuration/traffic-shaping.mdx

```diff
@@ -170,11 +172,19 @@ traffic_shaping:
 
 ### HTTP/2
 
-The router supports subgraph connections over
-- HTTP/2 with TLS. This is the default configuration.
+The router supports subgraph connections over:
+- HTTP/1.1
+- HTTP/1.1 with TLS
+- HTTP/2 with TLS
 - HTTP/2 Cleartext protocol (h2c). This uses HTTP/2 over plaintext connections.
```

Comment:
From what I've seen `H2C` is more common than `h2c`, particularly if we're going with the more formal HTTP capitalization (vs http).

Reply:
For whatever reason, the authoritative [IETF RFC](https://datatracker.ietf.org/doc/html/rfc7540#section-3.1) uses `HTTP` and `h2c`.

Reply:
Ah, interesting.